jenkins-pipelines/performance_staging/perf-regression-scylla-gradual-throughput-grow-uniform-2.3tb.jenkinsfile

```diff
@@ -7,7 +7,7 @@ perfRegressionParallelPipeline(
     backend: "aws",
     aws_region: "us-east-1",
     test_name: "performance_regression_gradual_grow_throughput.PerformanceRegressionGradualGrowThroughutTest",
-    test_config: '''["test-cases/performance/perf-regression-gradual-throughput-grow.yaml", "configurations/cs-gauss-distribution.yaml"]''',
+    test_config: '''["test-cases/performance/perf-regression-gradual-throughput-grow-2.3tb.yaml", "configurations/cs-uniform-distribution.yaml", "configurations/disable_kms.yaml", "configurations/tablets_disabled.yaml"]''',
```

Comment:
do we need this pipeline?

Reply:
this is old pipeline that I did not worked on it. Please, write the exactly name. If I remove "uniform" it will remain with "grow" in the end. If remove the "grow" (as you asked above) it will be with the same name as new pipeline.

Reply:
If it's the old pipeline and we leave this code, leave it as it was without changing anything.